spec/processors/jpeg2k_spec.rb

```diff
@@ -54,4 +56,25 @@
       expect(r).to eq(described_class.calculate_recipe(args, 'grey', 7200))
     end
   end
+
+  describe "#encode" do
+    it "executes the external utility" do
+      expect(described_class).to receive(:execute) { 0 }
+      described_class.encode('infile', 'recipe', 'outfile')
+      # expect(Hydra::Derivatives::Processors::ShellBasedProcessor).to have_received(:execute)
```

Comment:
Please remove the commented out code. It looks like this is tested in the above `expect` segment.